Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
`f.tap?` and `f.tap` would do the same regex matching twice. So squash them into one in the benefit of performance.